daemon/daemon.go

```diff
@@ -66,6 +66,9 @@ var (
 	// containerd if none is specified
 	DefaultRuntimeBinary = "docker-runc"
 
+	// DefaultInitBinary is the name of the default init binary
+	DefaultInitBinary = "docker-init"
```

Comment:
Minor nit, but perhaps this should be in platform files with a comment that it's not used on Windows.